router/http_test.go

```diff
@@ -1005,6 +1004,7 @@ func (s *S) TestDefaultServerKeypair(c *C) {
 	assertGet(c, "https://"+l.TLSAddr, "foo.example.com", "2")
 }
 
+// TODO(bgentry): validate this in Postgres??
 func (s *S) TestCaseInsensitiveDomain(c *C) {
```

Comment:
We're still establishing conventions. One layer where we should apply it is via the JSON schemas. I think they exist for the router, so we should probably just open a ticket for adding them to the API. The other place that might make sense is as a postgres constraint, but that doesn't seem to important as all writes will go through the API.